api/services/containers/v1/containers.proto

```diff
@@ -82,6 +82,11 @@ message Container {
 
 	// UpdatedAt is the last time the container was mutated.
 	google.protobuf.Timestamp updated_at = 9 [(gogoproto.stdtime) = true, (gogoproto.nullable) = false];
+
+	// Extensions is extra/misc data used by the client.
```

Comment:
"Extensions allow clients to provide zero or more blobs that are directly associated with the container. One may provide protobuf, json or other encoding formats. The primary use of this is to further decorate the container object with field that may be specific to a client integration."